### PR TITLE
docs: vue examples

### PR DESCRIPTION
**Pull request by kouts**
Thanks for this great library and for making the Tanstack framework-agnostic! 
This PR adds the following examples for Vue:
- [dynamic](https://codesandbox.io/p/sandbox/tanstack-vue-virtual-example-dynamic-mc4k9q)
- [fixed](https://codesandbox.io/p/sandbox/tanstack-vue-virtual-example-fixed-4kg2cg)
- [infinite-scroll](https://codesandbox.io/p/sandbox/tanstack-vue-virtual-example-infinite-scroll-q5s3hq)
- [smooth-scroll](https://codesandbox.io/p/sandbox/tanstack-vue-virtual-example-smooth-scroll-qcjrtl)
- [sticky](https://codesandbox.io/p/sandbox/tanstack-vue-virtual-example-sticky-qkcmvc)
- [table](https://codesandbox.io/p/sandbox/tanstack-vue-virtual-example-table-v2m8jd)
- [variable](https://codesandbox.io/p/sandbox/tanstack-vue-virtual-example-variable-9td9dp)

The `padding` examples are still pending, but I believe this is a good start.

PS: I tried running the docs locally but couldn't figure it out, it would be nice if we can have some instructions on how to do that.

### CODE REVIEW

File: examples/vue/table/src/App.vue [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+  getCoreRowModel,
+  getSortedRowModel,
+} from '@tanstack/vue-table'
+import { makeData, Person } from './makeData'
+
+const data = ref(makeData(50_000))
+
+const sorting = ref<SortingState>([])
+
+const getSortingHandler = (e: Event, fn: any) => {
```
Comment (kouts) [inline examples/vue/table/src/App.vue:88]: I couldn't figure out the `type` for this function here.

File: examples/vue/table/src/App.vue [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  return fn(e)
+}
+
+const setSorting = (sortingUpdater: any) => {
+  const newSortVal = sortingUpdater(sorting.value)
+
+  sorting.value = newSortVal
+}
```
Comment (kouts) [inline examples/vue/table/src/App.vue:96]: Same here, I couldn't figure out the `type` for the `sortingUpdater` function.

File: examples/react/table/src/main.tsx [hunk 3]
```diff
@@ -26,12 +26,12 @@ function ReactTableVirtualized() {
       },
```
Comment (kouts) [inline examples/react/table/src/main.tsx:26]: This file only contains formatting changes due to prettier being applied.

File: package.json [hunk 4]
```diff
@@ -11,7 +11,7 @@
     "linkAll": "lerna exec 'npm link' --parallel",
     "unlinkAll": "lerna exec 'npm unlink' --parallel",
     "dev": "npm run clean && npm run watch",
-    "prettier": "prettier \"packages/*/{src/**,examples/**/src/**}.{md,js,jsx,ts,tsx,json}\" --write",
+    "prettier": "prettier \"{packages/*/src/**,examples/*/*/src/**}.{md,js,jsx,ts,tsx,vue,json}\" --write",
```
Comment (kouts) [inline package.json:14]: Added the `.vue` extension and fixed the `prettier` command `glob` so that it takes into account the examples as well.

**Comment by piecyk** [comment]
Thanks @kouts

**Pull request merged**